testing/tests/DevExpress.ui.widgets.editors/colorBox.tests.js

```diff
@@ -396,7 +396,7 @@ QUnit.test("In 'instantly' mode popup should not disappear if value was changed"
         top: 80
     });
 
-    assert.equal($overlay.css("display"), "flex");
+    assert.equal($overlay.css("display"), "block");
```

Comment:
This assertion is not linked with test name. It can lead to the question: "Why should I use exactly display: block for preventing popup disappear?"